fp.js

```diff
@@ -18,17 +24,36 @@ var people = [
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
+/*
 people.forEach(function(person) {
   if(person.age < 35){
     peopleYoungerThan35.push(person)
   }
 });
+*/
+ people.filter(people=>{
+    if(people.age < 35){
+      peopleYoungerThan35.push(people);
+    }
+ });
+ console.log(people);
+ console.log(peopleYoungerThan35);
 
 // Convert the following to use `.forEach`
 
+ var peopleWhoseNamesHaveTheLetterS = [];
 var people = [
   {name: "Jane", age: 32},
   {name: "Jessica", age: 34},
   {name: "Jasmine", age: 39}
 ]
-var peopleWhoseNamesHaveTheLetterS = people.filter(function(person){return person.name.match("s")});
+  //var peopleWhoseNamesHaveTheLetterS = people.filter(function(person){return person.name.match("s")});
+
+    people.forEach(people=>{
+    if(people.name.match("s")){
+      return peopleWhoseNamesHaveTheLetterS.push(people);
```

Comment:
:+1: Although you don't need to use the `return` keyword here

Also, you should not use `people` as the argument for your callback. Instead, use something that represents each item in the `people` array -- like `person`.